docs/version0.35.md

```diff
@@ -32,6 +32,7 @@ The following new features and notable changes since version 0.33.1 are included
 - [New `user2` event added for the `-Xdump` option](#new-user2-event-added-for-the-xdump-option)
 - [New `-XX:[+|-]PerfTool` option added](#new-xx-perftool-option-added)
 - [New default options added in the `options.default` file](#new-default-options-added-in-the-optionsdefault-file)
+- [Options added to encrypt the JITServer metrics](#options-added-to-encrypt-the-jitserver-metrics)
```

Comment:
Since all the bullets above start with "New" maybe we should do here the same for consistency.

Reply:
Done.